tests/hpc/slurm_master_db.pm

```diff
@@ -53,39 +212,29 @@ sub run {
     barrier_wait("SLURM_SLAVE_SERVICE_ENABLED");
 
     $self->check_nodes_availability();
-    assert_script_run("srun -w slave-node00 date");
 
-    ## TODO provide more comprehensive checks for slurm
-    # job scheduling
-    for (my $i = 0; $i < 20; $i++) {
-        assert_script_run("srun -w slave-node00 date");
-        assert_script_run("srun -w slave-node01 date");
-    }
+    my $file = 'tmpresults.xml';
+    assert_script_run("touch $file");
+    script_run("echo \"<testsuite name='HPC single tests'>\" | tee $file");
 
-    ## test sbatch
-    my $sbatch = 'slurm_sbatch.sh';
-    script_run("wget --quiet " . data_url("hpc/$sbatch") . " -O $sbatch");
-    assert_script_run("chmod +x $sbatch");
-    record_info('meminfo', script_output("cat /proc/meminfo"));
-    script_output("sbatch $sbatch");
-    ##sbatch SBATCH --time=0-00:01:00
-    ## so the worker should wait for the sbatch to finish
-    sleep(70);
-    upload_logs('/tmp/sbatch1');
-    record_info('INFO', 'sbatch slurm_sbatch.sh');
+    my %test;
+    %test = test_01();
+    pars_results(%test);
 
-    ## test slurm-torque: https://fate.suse.com/323998
-    my $pbs = 'slurm_pbs.sh';
-    script_run("wget --quiet " . data_url("hpc/$pbs") . " -O $pbs");
-    assert_script_run("chmod +x $pbs");
-    script_output("sbatch $pbs");
-    ## execution (wall time) time set to 1m and there is a sleep
-    ## in the PBS script
-    ## so the worker should wait for the pbs to finish
-    sleep(80);
-    upload_logs('/tmp/Job_PBS_o');
-    upload_logs('/tmp/Job_PBS_e');
-    record_info('INFO', 'sbatch slurm_pbs.sh');
+    %test = test_02();
+    pars_results(%test);
+
+    #%test = test_03();
```

Comment:
commented code should be deleted before commit

Reply:
removed.